rackcity/views/it_model_views.py

```diff
@@ -18,22 +18,33 @@ def model_list(request):
         return JsonResponse(serializer.data, safe=False)
 
 
-def model_add(request):  # TODO: make it only accept POSTs
+@api_view(['POST'])
+@permission_classes([IsAdminUser])
+def model_add(request):
     """
     Add a new model
     """
-    if request.method == 'POST':
-        data = JSONParser().parse(request)
-        serializer = ITModelSerializer(data=data)
-        if serializer.is_valid():  # TODO: figure out what .valid() does.  Different fields are required for add, update, etc
-            print("valid")
+    data = JSONParser().parse(request)
+    failure_message = ""
+    if 'id' in data:
+        failure_message = failure_message + "Don't include id when adding a model. "
+    serializer = ITModelSerializer(data=data)
+    if not serializer.is_valid(raise_exception=False):
+        failure_message = failure_message + str(serializer.errors)
+    if failure_message == "":
+        try:
             serializer.save()
-            # should return something better maybe
-            return HttpResponse(status=201)
-        else:
-            print("invalid")
-            # should return something better definitely
-            return HttpResponse(status=400)
+            return JsonResponse({"success": True}, status=201)
```

Comment:
I made this 201 "Created" but we can change it if it's difficult for front end

Reply:
this may be too picky of me but I think we should use python's http library (`from http import HTTPStatus`) then replace `status=201` with `status=HTTPStatus.CREATED`

I think that makes the code more readable instead of having to remember the nuances between various 2xx, 4xx, 5xx errors

Reply:
another thing - do we still need "success" in the response data or is that redundant? I think we could just send empty data dict for 200s

Reply:
I changed this to be an HttpResponse since it doesn't have a body